source/cep/cep3.rst

```diff
@@ -339,10 +339,9 @@ The command for generating |cyclus| specific nuclear data is as follows:
 
 Once the file is generated it must be put onto rackspace.
 
-**Update Gtest:** We include a copy of the fused Gtest source code within our
-source tree located in the ``tests/GoogleTest`` directory.  To keep up with
-Gtest's natural evolution cycle, please download the latest release of Google Tests
-and follow `the fused source directions here`_.  If we go too long without doing this,
+**Update GTest:** We fetch the GTest source code during our CMake build.  To keep up with
+GTest's natural evolution cycle, please reference the latest release of Google Tests
+in ``tests/CMakeLists.txt``  If we go too long without doing this,
```

Comment:
This looks good! I have a question that would help me clarify for my review: 
- what needs to be updated in `tests/CMakeLists.txt`? Is it just a single line? Or will several changes need to happen? Maybe it would be clearer to quote the actual argument that needs to be updated in the file?

Reply:
I just added the command that needs to be updated.  Let me know if you think this is sufficient instruction.